qc_grader/challenges/fall_2022/lab3.py

```diff
@@ -1,33 +1,28 @@
 from typeguard import typechecked
+
+from typing import Callable
+
 from qiskit import QuantumCircuit
+from qiskit.result import QuasiDistribution
 from qc_grader.grader.grade import grade, get_problem_set
+import random
 
 _challenge_id = 'fall_2022'
 
 @typechecked
-def grade_lab3_ex1(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-1', _challenge_id)
-
-@typechecked
-def grade_lab3_ex2(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-2', _challenge_id)
-
-@typechecked
-def grade_lab3_ex3(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-3', _challenge_id)
-
-@typechecked
-def grade_lab3_ex4(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-4', _challenge_id)
-
-@typechecked
-def grade_lab3_ex5(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-5', _challenge_id)
+def grade_lab3_ex1(attempt_qc: QuantumCircuit) -> None:
+    grade(QuantumCircuit, 'ex3-1', _challenge_id)
```

Comment:
this should be passing the object (`attempt_qc`) instead of the class (`QuantumCircuit`)